osu.Game/IO/OsuStorage.cs

```diff
@@ -33,12 +33,17 @@ public class OsuStorage : MigratableStorage
         private readonly StorageConfigManager storageConfig;
         private readonly Storage defaultStorage;
 
-        public override string[] IgnoreDirectories => new[] { "cache" };
+        public override string[] IgnoreDirectories => new[]
+        {
+            "cache",
+            "client.realm.management"
+        };
 
         public override string[] IgnoreFiles => new[]
         {
             "framework.ini",
-            "storage.ini"
+            "storage.ini",
+            "client.realm.note"
```

Comment:
i think the `.note` file was specifically important as it caused errors when attempting to move (was a socket or something?)

Reply:
Yeah it was, and I've got nothing against it. I'd mostly just put the lockfile in here too, since in the context of a possible data migration copying it might yield a bit of a broken state if it is copied over.